src/app/(rucio)/did/list/page.tsx

```diff
@@ -15,7 +15,7 @@ export default function Page() {
     }
     const didQuery = async (query: string, type: DIDType) => {
         const request: any = {
-            url: new URL(`${process.env.NEXT_PUBLIC_WEBUI_HOST}/api/feature/list-dids`),
```

Comment:
This change refers to making deployments more flexible and accessible through any hostname. All the places where the hostname was specified through an environmental variable are reviewed.